packages/core/src/__tests__/rum/instrumentation/DdRumErrorTracking.test.tsx

```diff
@@ -13,6 +13,8 @@ import type {
 import { DdRumErrorTracking } from '../../../rum/instrumentation/DdRumErrorTracking';
 import { BufferSingleton } from '../../../sdk/DatadogProvider/Buffer/BufferSingleton';
 
+jest.mock('../../../utils/jsUtils');
```

Comment:
Curious why this needed to be added?

Reply:
I don't want to add a 50ms delay in all my tests here - it will make them harder to debug in case they go wrong.

I added a default mock under `utils/__mocks__/jsUtils.ts` that is used when I add this line in a test file.